optuna/samplers/tpe/sampler.py

```diff
@@ -534,9 +535,25 @@ def _get_observation_pairs(study, param_name):
     if study.direction == StudyDirection.MAXIMIZE:
         sign = -1
 
+    _study = study
```

Comment:
Can you simply overwrite `study` instead of declaring a new `_study`?

Reply:
Sure.